ocf/ocf.go

```diff
@@ -53,8 +53,24 @@ type Decoder struct {
 	count int64
 }
 
+type decoderConfig struct{}
+
+// DecoderFunc represents a configuration function for Decoder.
+type DecoderFunc func(*decoderConfig)
+
+// WithSkipNameValidation sets whether to skip validating of field names when decoding.
+func WithSkipNameValidation(skip bool) DecoderFunc {
```

Comment:
I would urge caution here. It is unclear that this is setting a global value, so creating 2 decoders, one with this option and one without, it would be expected that they behave differently, however they will not. I would suggest leaving this out.

Reply:
Perhaps in the second option the name could be something like GlobalSkipNameValidation, but yeah, a global var setting would do the same semantically. 

I'll remove it

Reply:
Done. Please take another look